libs/blocks/table/table.js

```diff
@@ -153,6 +153,15 @@ function handleAddOnContent(table) {
   table.addEventListener('mas:resolved', debounce(() => { handleEqualHeight(table, '.row-heading'); }));
 }
 
+async function setAriaLabelsForExpandableIcons() {
+  const config = getConfig();
+  const ariaLabel = await replaceKey('toggle-row', config);
+  const icons = document.querySelectorAll('.icon.expand[role="button"]');
```

Comment:
Although I agree that all expand icons should get the `aria-label` attribute, I'm not sure this is the place to do it. I'd either apply the behavior in a more general icon file or scope these changes to just the table we're decorating.